### DISCUSSION TITLE
Differences in parsing between show and uninstall?

### DISCUSSION THREAD

**Discussion by ternarydreams** [microsoft/winget-cli#3719] [Q&A]
Is there an error between the keyboard and chair or is this behavior just inconsistent? I should be able to find a package with the same string I use to uninstall it. 

![confusingwingetbehavior](https://github.com/microsoft/winget-cli/assets/77955661/9f6a896c-4d15-4ab8-bf55-e60da272d9f6)

**Comment by denelon** [comment]
The uninstall can use many different fields to identify packages that don't have a corresponding manifest in one of the configured sources for WinGet. The "show" command only works to display metadata from a manifest in a configured source when there is a matching installed package if the "Name" is being passed as the argument.
